cli/main.rs

```diff
@@ -93,6 +94,40 @@ use std::rc::Rc;
 use std::sync::Arc;
 use upgrade::upgrade_command;
 
+impl DocFileLoader for SourceFileFetcher {
```

Comment:
There's already an implementation called `DocLoader` - prefer to use it with import map set to `None`